File: content/problems/1945-sum-of-digits-of-string-after-convert/solutions/direct-first-digit-sum/Solution.java
class Solution {
    public int getLucky(String s,int k){
        int value=0;
        for(char letter:s.toCharArray()){
            int number=letter-'a'+1;
            value+=number/10+number%10;
        }
        for(int round=1;round<k;round++){
            int next=0;
            while(value>0){
                next+=value%10;
                value/=10;
            }
            value=next;
        }
        return value;
    }
}

File: content/problems/1945-sum-of-digits-of-string-after-convert/solutions/direct-first-digit-sum/Solution.kt
class Solution {
    fun getLucky(s:String,k:Int):Int{
        var value=0;
        for(letter in s){
            val number=letter-'a'+1;
            value+=number/10+number%10
        };
        repeat(k-1){
            var next=0;
            while(value>0){
                next+=value%10;
                value/=10
            };
            value=next
        };
        return value
    }
}
